src/arrayMethodSort.js

```diff
@@ -3,9 +3,25 @@
 /**
  * Implement method Sort
  */
+
+const compareStr = (a, b) => `${a}` > `${b}` ? 1 : -1;
```

Comment:
But what if a == b? In this case they are equal and we don't need to swap anything. It's a usual case to return 0, -1 or 1 from the comparison function